pkg/aggregator/aggregator.go

```diff
@@ -765,8 +802,19 @@ func (agg *BufferedAggregator) run() {
 					log.Errorf("Error submitting orchestrator data: %s", err)
 				}
 			}(orchestratorMetadata)
-		}
+		case event := <-agg.eventPlatformIn:
```

Comment:
Not sure here. Is there a need to add `tlmProcessed.Inc("event_platform_events")` ?

Reply:
Yeah we can do that. Added `tlmProcessed.Add(1, event.eventType)` as we're tracking by eventType. 

It's kind of redundant as we're already tracking the flushes, but I guess might as well track in `processed` for consistency.